src/app/views/cheat-sheets/balancers/balancers.component.html

```diff
@@ -33,24 +33,22 @@
         <div class="col-12 col-md-5">
             <strong class="d-block text-center">Tips</strong>
             <ul>
-                <li>Don't overuse balancers, use when needed.</li>
+                <li>Don't overuse balancers, use them when needed.</li>
                 <li>
-                    It's generally recommended to make builds inherently balanced targeting a complete use of a lane.
+                    Make a build that's inherently balanced, by letting the assembly machines consume a full belt of items.
                 </li>
                 <li>
                     Balancers can still be useful regardless; most common are the
                     <app-blueprint-clipboard [url]="sheetData?.commonBalancers[0].raw"></app-blueprint-clipboard>
                     <a href="{{sheetData?.commonBalancers[0].url}}" target="_blank" rel="noopener">4<i class="fas fa-long-arrow-alt-right"></i>4</a> and
                     <app-blueprint-clipboard [url]="sheetData?.commonBalancers[1].raw"></app-blueprint-clipboard>
-                    <a href="{{sheetData?.commonBalancers[1].url}}" target="_blank" rel="noopener">8<i class="fas fa-long-arrow-alt-right"></i>8</a> throughput unlimited.
+                    <a href="{{sheetData?.commonBalancers[1].url}}" target="_blank" rel="noopener">8<i class="fas fa-long-arrow-alt-right"></i>8</a> throughput unlimited varieties.
```

Comment:
Maybe remove the " throughput unlimited varieties." since you elaborate after

Reply:
I'm not sure that I understand what you want me to do...

1. Do you want me to remove this line?
>Being throughput unlimited input/output, the amount can vary thus those two cover most use cases.

2. Or do you want me to remove the second link to the 8x8 balancers (_and remove the entry from balancers.json_) `<a href="{{sheetData?.commonBalancers[1].url}}" target="_blank" rel="noopener">8<i class="fas fa-long-arrow-alt-right"></i>8</a>`?

3. Or do you want me to remove this text?
> throughput unlimited varieties

Reply:
Also read https://github.com/deniszholob/factorio-cheat-sheet/pull/268#issuecomment-674359617 for tips on how to apply the change from within the Pull Request interface as a maintainer of the project. 😉
I think you'll like it.